plugins/uefi-capsule/fu-uefi-capsule-plugin.c

```diff
@@ -619,6 +619,24 @@ static FuVolume *
 fu_uefi_capsule_plugin_get_default_esp(FuPlugin *plugin, GError **error)
 {
 	g_autoptr(GPtrArray) esp_volumes = NULL;
+	const gchar *recovery_partitions[] = {
+	    "DellRestore",
+	    "DellUtility",
+	    "DIAGS",
+	    "HP_RECOVERY",
+	    "IBM_SERVICE",
+	    "IntelRST",
+	    "Lenovo_Recovery",
+	    "Recovery",
+	    "RECOVERY",
+	    "Recovery_Partition",
+	    "SERVICEV001",
+	    "SERVICEV002",
+	    "System_Reserved",
+	    "SYSTEM_RESERVED",
+	    "WINRE_DRV",
+	    NULL,
```

Comment:
even though it's missing from that list in udisks, I think you should add `OS` to this list.  I found it comes specifically in the case of recovery partition created by dell-recovery media:

https://github.com/dell/dell-recovery/blob/master/ubiquity/dell-bootstrap.py#L1091

Reply:
Gotcha, thanks. Added.